src/pages/Group/GroupStatistics.tsx

```diff
@@ -0,0 +1,262 @@
+import { useEffect, useState } from "react";
+import { useQuery, useQueryClient } from "react-query";
+import { useLocation } from "react-router-dom";
+import ExpandLeftIcon from "../../assets/ExpandLeftIcon";
+import ExpandRightIcon from "../../assets/ExpandRightIcon";
+import DateRangeSwitcher from "../../components/DateRangeSwitcher";
+import Graph from "../../components/Graph";
+import HappinessCalendar from "../../components/HappinessCalendar";
+import Row from "../../components/layout/Row";
+import GroupHappinessModal from "../../components/modals/GroupHappinessModal";
+import { QueryKeys } from "../../constants";
+import { useApi } from "../../contexts/ApiProvider";
+import { useUser } from "../../contexts/UserProvider";
+import { Happiness } from "../../data/models/Happiness";
+import { formatDate, useWindowDimensions } from "../../utils";
+import Stat from "../Statistics/Stat";
+import { Group } from "../../data/models/Group";
+import { format } from "mathjs";
+import HappinessViewerModal from "../../components/modals/HappinessViewerModal";
+
+/**
+ * The page for displaying statistics for the current user
```

Comment:
also if we are keeping this as a separate page you will need to update this comment.